runtime/src/main/java/org/corfudb/security/sasl/plaintext/PlainTextLoginModule.java

```diff
@@ -3,19 +3,19 @@
 import java.io.IOException;
 import java.util.Map;
 
+import javax.security.auth.Subject;
 import javax.security.auth.callback.Callback;
+import javax.security.auth.callback.CallbackHandler;
 import javax.security.auth.callback.NameCallback;
 import javax.security.auth.callback.PasswordCallback;
-import javax.security.auth.callback.CallbackHandler;
 import javax.security.auth.callback.UnsupportedCallbackException;
 import javax.security.auth.login.LoginException;
 import javax.security.auth.spi.LoginModule;
-import javax.security.auth.Subject;
 
 /**
  * Created by sneginhal on 01/27/2017
  *
- * Implementation of the plain text LoginMoodule.
+ * <p>Implementation of the plain text LoginMoodule.
```

Comment:
@no2chem  According to http://www.oracle.com/technetwork/articles/java/index-137868.html, the doc text is HTML (some closing tags are optional, like p's) and not XHTML (no optional closing tags).

I put the <p> at the start of the next paragraph.  The Oracle doc examples putting it on the "blank" line.  The Google style guide https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs says to use the style I've used here.  It also says nothing about XHTML compliance or HTML optional tags, alas.

Thoughts?